routes/routes.js

```diff
@@ -68,4 +86,23 @@ router.get('/error', (req, res) =>{
   res.render('pages/error')
 })
 
-module.exports = router;
\ No newline at end of file
+module.exports = router;
+
+router.use(router);
```

Comment:
LINES 93 - 108 will need to be removed because this is the server which is already on the index page. However, LINES 96-107 Should be moved to index.js and within the server code.